resources/scss/ceres/widgets/Grid/_grid-widget.scss

```diff
@@ -8,7 +8,7 @@
         display: flex;
     }
 
-    .widget-inner
+    > .widget-inner
```

Comment:
wozu wird das in diesem Zusammenhang benötigt? Die Grid-Widgets werden hier (https://github.com/plentymarkets/plugin-ceres/pull/790) komplett überarbeitet

Reply:
Die Last Seen Liste hat hierdurch 30px zu viel Padding bekommen. Wenn du das überarbeitet hast nehme ich das erst mal raus und wir schauen wie es sich dann mit deinen Änderungen verhält und ändern es ggf nachträglich.